src/components/Header.tsx

```diff
@@ -127,8 +129,33 @@ const Menu = styled.div`
   align-items: center;
 `;
 
+const ToolTip = styled.span`
+  visibility: hidden;
+  width: 120px;
+  text-align: center;
+  border-radius: 8px;
+  padding: 5px 0;
+  position: absolute;
+  z-index: 1;
+  top: 100%;
+  color: white;
+  background-color: black;
+
+  &::after {
+    content: "";
+  position: absolute;
+  bottom: 100%;
+  left: 50%;
+  margin-left: -5px;
+  border-width: 5px;
+  border-style: solid;
+  border-color: transparent transparent black transparent;
```

Comment:
Indentation up here is off as well. I would suggest using prettier + maybe the Styled Component extension for VSCode (if you're using VSCode) so that it will fix all automatically :)